docs/api.md

```diff
@@ -2170,22 +2214,6 @@ Identifies what kind of target this is. Can be `"page"`, `"service_worker"`, or
 > **NOTE** JavaScript Coverage doesn't include anonymous scripts; however, scripts with sourceURLs are
 reported.
```

Comment:
What happens if there is an injected inline style?

Reply:
The style will be ignored similarly to the anonymous scripts in js coverage. Added a test and a doc note for this.